guides/common/modules/con_configuring-project-server-with-an-http-proxy.adoc

```diff
@@ -0,0 +1,6 @@
+:_mod-docs-content-type: CONCEPT
+
+[id="configuring-{project-context}-server-with-an-http-proxy"]
+= Configuring {ProjectServer} with an HTTP proxy
+
+Use the following procedures to configure {ProjectServer} with an HTTP proxy.
```

Comment:
Avoid self-referential content. Instead we could say where the HTTP proxy is located - internet-ProjectServer or ProjectServer-hosts?

For example:
```suggestion
Configure {ProjectServer} with an HTTP proxy if you need your hosts to access {ProjectServer} through the HTTP proxy.
```

Reply:
This wording came from the original assembly. I reworded it slightly, though it still feels a bit awkward to me.